populate-metadata/src/extension.ts

```diff
@@ -0,0 +1,62 @@
+// Documentation: https://sdk.netlify.com
+import {
+  type BuildHookOptions,
+  type BuildHookType,
+  type BuildHookWithContext,
+  NetlifyExtension,
+} from '@netlify/sdk';
+import type z from 'zod';
+
+export const envVarToBool = (envVar: boolean | string = 'false') => {
```

Comment:
Declare return types on exported functions
```suggestion
export const envVarToBool = (envVar: boolean | string = 'false'): boolean => {
```